### PR TITLE
Update RedBag's Solitude

### PR DESCRIPTION
**Pull request by robertgk2017**
and some formatting consistency changes.  Condition needed for the SPO patch??

### CODE REVIEW

**Comment by robertgk2017** [comment]
Through this in without any conditions on the spo patch, wondering what you think it ought to be if any at all?

**Comment by robertgk2017** [comment]
hang on im fixing that bad check

**Comment by robertgk2017** [comment]
passed checks, what you think for the spo patch, it points to the 1.5 file which seems to be the main file that is being downloaded today as a master.

**Comment by sibir-ine** [comment]
The patch has `Skyrim Project Optimization - Full ESL Version.esm` as a master, so a message for it should only be displayed if that plugin is active.

**Comment by robertgk2017** [comment]
Sounds good to me lemme throw that in there.  Should i do 3rdparty or included msg keys?  since the patch is on my fixes page

File: masterlist.yaml [hunk 1]
```diff
@@ -21262,13 +21262,27 @@ plugins:
...
+        name: 'Rob''s Bug Fixes - RedBag''s Solitude'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
+        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
+    clean:
+      - crc: 0xA9A23AB8
+        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+      - crc: 0x90AD69F2
+        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```
Comment (sibir-ine) [inline masterlist.yaml]: The `util` string for clean cleaning data doesn't support Markdown (only dirty ones do), you can just copy & paste what xEdit reports after QAC:

```suggestion
      - crc: 0xA9A23AB8
        util: 'SSEEdit v4.0.4b'
      - crc: 0x90AD69F2
        util: 'SSEEdit v4.0.4b'
```

**Comment by sibir-ine** [comment]
Either is fine, but `patchIncluded` would need to check for your replacer plugin.

**Comment by robertgk2017** [comment]
Went for 3rd party cuz it doesnt specifically need my file.

File: masterlist.yaml [hunk 2]
```diff
@@ -21262,13 +21262,32 @@ plugins:
...
+  - name: 'RedBag''s Solitude - SPO Patch.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64068/'
+        name: 'Rob''s Bug Fixes - RedBag''s Solitude'
+    clean:
+      - crc: 0xB4B8EE8D
+        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - crc: 0xB4B8EE8D
        util: 'SSEEdit v4.0.4b'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -21262,13 +21262,32 @@ plugins:
...
+        name: 'Rob''s Bug Fixes - RedBag''s Solitude'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
+        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Project Optimization'
+          - '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)'
+        condition: 'active("Skyrim Project Optimization - Full ESL Version.esm") and not active("RedBag''s Solitude - SPO Patch.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]: Looking at the patch in xEdit, it's only necessary for your replacer plugin, so `patchIncluded` would actually be more appropriate. The messages should also be sorted alphabetically by alias:

```suggestion
      - <<: *patchIncluded
        subs: [ 'Skyrim Project Optimization' ]
        condition: 'active("Skyrim Project Optimization - Full ESL Version.esm") and checksum("RedBag''s Solitude.esp", 90AD69F2) and not active("RedBag''s Solitude - SPO Patch.esp")'
      - <<: *patchUpdateAvailable
        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Review by sibir-ine — APPROVED** [review]
Thanks!

File: masterlist.yaml [hunk 4]
```diff
@@ -21262,13 +21262,30 @@ plugins:
...
 
   - name: 'RedBag''s Solitude.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xC025F482
-        util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 95
-        udr: 79
+    url:
```
Comment (sibir-ine) [inline masterlist.yaml]: Sorry, there are a few more minor issues I almost missed:

```suggestion
    url:
```

File: masterlist.yaml [hunk 5]
```diff
@@ -21262,13 +21262,30 @@ plugins:
...
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xC025F482
-        util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 95
-        udr: 79
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/'
+        name: 'Redbag''s Solitude'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        name: 'RedBag''s Solitude'
```

File: masterlist.yaml [hunk 6]
```diff
@@ -21262,13 +21262,30 @@ plugins:
...
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
+        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
+    clean:
+      - crc: 0xA9A23AB8
+        util: 'SSEEdit v4.0.4b'
+      - crc: 0x90AD69F2
+        util: 'SSEEdit v4.0.4b'
+  - name: 'RedBag''s Solitude - SPO Patch.esp'
+    url:
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
    url:
```

**Pull request merged**